apis/Google.Cloud.AspNetCore.DataProtection.SecretManager/.repo-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "distribution_name": "Google.Cloud.AspNetCore.DataProtection.SecretManager",
+    "release_level": "stable",
```

Comment:
Couple of questions:

.NET PowerPack repository was released as alpha, we are directly migrating that to a stable release?
  
I am not versed with the values of library_type. Could you please point me to the documentation you referred to provide the value as INTEGRATION (this is for my reference)

Reply:
yes @Rishabh-V on the release part I am also a bit nervous on that I think we can start up with `alpha`  first.

on the library_type part you can refer to [go/library-types](https://goto.google.com/library-types)

Reply:
Thanks. Reading the document, I am still not clear if `library_type` should be `INTEGRATION`. Let's get this reviewed from Jon and Amanda and we'll know if this the is right value to be used.